public/app/features/dashboard/utils/getPanelMenu.ts

```diff
@@ -43,39 +43,41 @@ export function getPanelMenu(
     locationService.partial({
       viewPanel: panel.id,
     });
+    reportInteraction('dashboards_panelheader_view_clicked');
   };
 
   const onEditPanel = (event: React.MouseEvent<any>) => {
     event.preventDefault();
     locationService.partial({
       editPanel: panel.id,
     });
+    reportInteraction('dashboards_panelheader_edit_clicked');
   };
 
   const onSharePanel = (event: React.MouseEvent<any>) => {
     event.preventDefault();
     sharePanel(dashboard, panel);
+    reportInteraction('dashboards_panelheader_share_clicked');
   };
 
   const onAddLibraryPanel = (event: React.MouseEvent<any>) => {
     event.preventDefault();
     addLibraryPanel(dashboard, panel);
+    reportInteraction('dashboards_panelheader_createlibrarypanel_clicked');
   };
 
   const onUnlinkLibraryPanel = (event: React.MouseEvent<any>) => {
     event.preventDefault();
     unlinkLibraryPanel(panel);
+    reportInteraction('dashboards_panelheader_unlinklibrarypanel_clicked');
   };
 
   const onInspectPanel = (tab?: InspectTab) => {
     locationService.partial({
       inspect: panel.id,
       inspectTab: tab,
     });
-
-    reportInteraction('grafana_panel_menu_inspect', {
-      tab: tab ?? InspectTab.Data,
-    });
```

Comment:
@axelavargas @thanos-karachalios we had this event already reporting. It uses `grafana_panel_menu_inspect` event name.

This was used to know if the Apply button is used. How is the impact of changing this? If I'm not wrong, we're the only ones using that event, so we can just update the queries using the new event name, or have 2 queries combined with the different names. I think, for consistency, we should use the same namespaces for all of the menu events: `dashboards_panelheader_<item>_clicked`

Reply:
As far as I remember we were the only ones using that event, but I am not sure if other teams are using it 😅